src/components/Sections/Events/Filters.js

```diff
@@ -15,12 +14,14 @@ import OriginalLanguageFilter from '../../FiltersAside/OriginalLanguageFilter/Or
 import TagSourceFilter from '../../FiltersAside/TopicsFilter/TagSourceFilter';
 import ContentTypesFilter from './ContentTypesFilter';
 
-const Filters = ({ namespace, baseParams, t }) => {
+const Filters = ({ namespace, baseParams }) => {
   const [isHydrated, setIsHydrated] = useState(false);
 
+  const { t }        = useTranslation();
   const isReady      = useSelector(state => selectors.isReady(state.filtersAside, namespace));
   const { wip, err } = useSelector(state => selectors.getWipErr(state.filtersAside, namespace));
-  const selected     = useSelector(state => filters.getNotEmptyFilters(state.filters, namespace), isEqual);
```

Comment:
Why did you remove here isEqual?

Reply:
not need this additional checking, I use Ref for compare with preview state